### DISCUSSION TITLE
Welcome to linktree Discussions!

### DISCUSSION THREAD

**Discussion by heysagnik** [heysagnik/Linkees#1] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by roopeshsn** [comment]
Hi @heysagnik The design by you was awesome. Why can't we create this as a no-code tool? So, any user can create his/her own linktree.

**Reply by heysagnik** [reply]
Had the idea . But being a one man army 🪖 it's very difficult for me to maintain work and studies. If the community help me then I can think of it.

**Reply by roopeshsn** [reply]
I am here to make this happen with you! We can discuss how we are going to build this along with the tech stack. Once decided we'll jump into the building process. I am happy to collab on this project 🤝 I'll connect with you via e-mail for further discussions.
